src/config/proxy.ts

```diff
@@ -0,0 +1,15 @@
+import envVars from './env';
+import {bootstrap} from 'global-agent';
+import Logger from 'bunyan';
+
+const logger = new Logger({name:'proxy'});
+
+if(envVars.PROXY){
+  logger.info(`configuring proxy: ${envVars.PROXY} for outbound calls`);
+  process.env.GLOBAL_AGENT_HTTP_PROXY = envVars.PROXY;
```

Comment:
Setting the environment variable required by `global-agent` from the "neutral" env var `PROXY`.